### ISSUE TITLE
PIP install to virtual environment fails

### ISSUE THREAD

**Issue by michaeljohnbarr** [omab/python-social-auth#177]
TLDR is "error: can't copy 'social\apps\django_app\default\tests.py': doesn't exist or not a regular file"

It seems that the symbolic link that is committed does not play well with virtual environments. When I download the tar.gz file and extract it, the symbolic link works as expected (this is on Windows). However, trying to install to the virtual environment yields the below error.

Install packages failed: Error occurred when installing package python-social-auth. 

The following command was executed:

packaging_tool.py install --build-dir C:\Users\Michael\AppData\Local\Temp\pycharm-packaging7430035467726148887.tmp python-social-auth

The error output of the command:

Downloading/unpacking python-social-auth
  Running setup.py (path:C:\Users\Michael\AppData\Local\Temp\pycharm-packaging7430035467726148887.tmp\python-social-auth\setup.py) egg_info for package python-social-auth

Downloading/unpacking requests>=1.1.0 (from python-social-auth)
Downloading/unpacking oauthlib>=0.3.8 (from python-social-auth)
  Running setup.py (path:C:\Users\Michael\AppData\Local\Temp\pycharm-packaging7430035467726148887.tmp\oauthlib\setup.py) egg_info for package oauthlib

Downloading/unpacking six>=1.2.0 (from python-social-auth)
  Downloading six-1.5.2-py2.py3-none-any.whl
Downloading/unpacking python3-openid>=3.0.1 (from python-social-auth)
  Running setup.py (path:C:\Users\Michael\AppData\Local\Temp\pycharm-packaging7430035467726148887.tmp\python3-openid\setup.py) egg_info for package python3-openid

```
warning: no files found matching 'NOTICE'
warning: no files found matching 'CHANGELOG'
warning: no files found matching 'README.md' under directory 'examples'
warning: no files found matching '*.css' under directory 'doc'
warning: no files found matching '*.html' under directory 'doc'
```

Downloading/unpacking requests-oauthlib>=0.3.0,<0.3.2 (from python-social-auth)
  Downloading requests-oauthlib-0.3.1.tar.gz
  Running setup.py (path:C:\Users\Michael\AppData\Local\Temp\pycharm-packaging7430035467726148887.tmp\requests-oauthlib\setup.py) egg_info for package requests-oauthlib

Installing collected packages: python-social-auth, requests, oauthlib, six, python3-openid, requests-oauthlib
  Running setup.py install for python-social-auth
    error: can't copy 'social\apps\django_app\default\tests.py': doesn't exist or not a regular file
    Complete output from command C:\Users\Michael\mvmstats\Scripts\python.exe -c "import setuptools, tokenize;**file**='C:\Users\Michael\AppData\Local\Temp\pycharm-packaging7430035467726148887.tmp\python-social-auth\setup.py';exec(compile(getattr(tokenize, 'open', open)(**file**).read().replace('\r\n', '\n'), **file**, 'exec'))" install --record c:\users\michael\appdata\local\temp\pip-53tlxu-record\install-record.txt --single-version-externally-managed --compile --install-headers C:\Users\Michael\mvmstats\include\site\python3.3:
    running install

running build

running build_py

creating build

creating build\lib

creating build\lib\social

copying social\actions.py -> build\lib\social

copying social\exceptions.py -> build\lib\social

copying social\p3.py -> build\lib\social

copying social\store.py -> build\lib\social

copying social\utils.py -> build\lib\social

copying social__init__.py -> build\lib\social

creating build\lib\social\apps

copying social\apps__init__.py -> build\lib\social\apps

creating build\lib\social\apps\django_app

copying social\apps\django_app\context_processors.py -> build\lib\social\apps\django_app

copying social\apps\django_app\middleware.py -> build\lib\social\apps\django_app

copying social\apps\django_app\tests.py -> build\lib\social\apps\django_app

copying social\apps\django_app\urls.py -> build\lib\social\apps\django_app

copying social\apps\django_app\utils.py -> build\lib\social\apps\django_app

copying social\apps\django_app\views.py -> build\lib\social\apps\django_app

copying social\apps\django_app__init__.py -> build\lib\social\apps\django_app

creating build\lib\social\apps\django_app\default

copying social\apps\django_app\default\admin.py -> build\lib\social\apps\django_app\default

copying social\apps\django_app\default\fields.py -> build\lib\social\apps\django_app\default

copying social\apps\django_app\default\models.py -> build\lib\social\apps\django_app\default

error: can't copy 'social\apps\django_app\default\tests.py': doesn't exist or not a regular file

---

Cleaning up...
Command C:\Users\Michael\mvmstats\Scripts\python.exe -c "import setuptools, tokenize;**file**='C:\Users\Michael\AppData\Local\Temp\pycharm-packaging7430035467726148887.tmp\python-social-auth\setup.py';exec(compile(getattr(tokenize, 'open', open)(**file**).read().replace('\r\n', '\n'), **file**, 'exec'))" install --record c:\users\michael\appdata\local\temp\pip-53tlxu-record\install-record.txt --single-version-externally-managed --compile --install-headers C:\Users\Michael\mvmstats\include\site\python3.3 failed with error code 1 in C:\Users\Michael\AppData\Local\Temp\pycharm-packaging7430035467726148887.tmp\python-social-auth
Storing debug log for failure in C:\Users\Michael\pip\pip.log
